### ISSUE TITLE
ERROR: to use other evaluation criteria, labels must be +1/-1

### ISSUE THREAD

**Issue by vincentjytsai-nycu** [cjlin1/libsvm#183]
Hi,

After adding the plug-in "eval.h" and "eval.cpp" used in crossvalidation with different criteria (AUC, F-score, etc.), there is an error when running "svm-predict" with heart_scale (the example data file):
ERROR: to use other evaluation criteria, labels must be +1/-1
![image](https://user-images.githubusercontent.com/89758515/164078427-6aca693f-3556-4c54-90c6-b072a4c3eaac.png)

I'm wondering what is the problem. Any help is appreciated. Thanks!

**Comment by cjlin1** [comment]
Did you also copy the new Makefile and rebuild the file?
I can run it without problem:

$ make
g++ -Wall -Wconversion -O3 -fPIC -c eval.cpp -o eval.o
g++ -Wall -Wconversion -O3 -fPIC svm-train.c svm.o eval.o -o svm-train 
-lm
g++ -Wall -Wconversion -O3 -fPIC svm-predict.c svm.o eval.o -o 
svm-predict -lm
$ ./svm-train heart_scale
*
optimization finished, #iter = 162
nu = 0.431029
obj = -100.877288, rho = 0.424462
nSV = 132, nBSV = 107
Total nSV = 132
$ ./svm-predict  heart_scale heart_scale.model o
Accuracy = 86.6667% (234/270) (classification)

**Comment by vincentjytsai-nycu** [comment]
I indeed copied the new Makefile and recompile the code. But the situation remains the same......
![image](https://user-images.githubusercontent.com/89758515/164107271-b76d1657-4102-4c8b-981b-c4f49b8f2be2.png)

**Comment by cjlin1** [comment]
Did you exactly follow what I did? Please show me the full log of what 
you did

**Comment by vincentjytsai-nycu** [comment]
1. I have updated the Makefile as the instruction says.
![image](https://user-images.githubusercontent.com/89758515/164108703-146f60fd-44b2-4e66-935a-b0dcce9e2f03.png)

2. I have modified svm-train.cpp and svm-predict following the instruction. The header file "eval.h" is also included.
![image](https://user-images.githubusercontent.com/89758515/164109296-e034cc45-255f-4d73-9b5d-a74da105c0ca.png)
![image](https://user-images.githubusercontent.com/89758515/164109408-78a5536e-a676-499d-9e61-b1bfbdd85fd9.png)

3. Then I compiled the code and executed the commands as you did, but the error still occurs......
![image](https://user-images.githubusercontent.com/89758515/164109761-c5eb6f89-6adb-4a23-83e7-16863c3f09ba.png)
